docs/language-usage.rst

```diff
@@ -0,0 +1,354 @@
+     ...     'new Android phone at the Consumer Electronic Show.  '
+     ...     'Sundar Pichai said in his keynote that users love '
+     ...     'their new Android phones.')
+     >>> document = client.document(text_content)
+
+By default, the document's language and type are English and
+plain text:
+
+  .. code-block:: python
+
+     >>> document.language
+     'en'
+     >>> document.doc_type
+     'PLAIN_TEXT'
+
+These values can also be specified in the constructor
+
+  .. code-block:: python
+
+     >>> html_content = """\
+     ... <html>
+     ...   <head>
+     ...     <title>Story Time</time>
+     ...   </head>
+     ...   <body>
+     ...     <p>La vaca salt&oacute; sobre la luna.</p>
+     ...   </body>
+     ... </html>
+     ... """
+     >>> document = client.document(html_content,
+     ...                            language='es',
+     ...                            doc_type=language.Document.HTML)
+
+The ``language`` arugment can be either ISO-639-1 or BCP-47 language
+codes; at the time, only English, Spanish, and Japanese are supported.
+However, the ``analyzeSentiment`` method only supports English text.
+
+The document type (``doc_type``) argument can be one of
+:attr:`Document.PLAIN_TEXT <gcloud.language.document.Document.PLAIN_TEXT>` or
+:attr:`Document.HTML <gcloud.language.document.Document.HTML>`.
+
+In addition to supplying the text / HTML content, a document can refer
+to content stored in `Google Cloud Storage`_:
+
+  .. code-block:: python
+
+     >>> document = client.document(bucket='my-text-bucket',
+     ...                            blob='sentiment-me.txt')
+     >>> document.gcs_url
+     'gs://my-text-bucket/sentiment-me.txt'
+
+.. _analyzeEntities: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeEntities
+.. _analyzeSentiment: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeSentiment
+.. _annotateText: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/annotateText
+.. _Document: https://cloud.google.com/natural-language/reference/rest/v1beta1/Document
+.. _Google Cloud Storage: https://cloud.google.com/storage/
+
+Analyze Entities
+----------------
+
+The :meth:`~gcloud.language.document.Document.analyze_entities` method
+finds named entities (i.e. proper names) in the text, entity types,
+salience (prominence), mentions for each entity, and other properties.
+
+  .. code-block:: python
+
+     >>> text_val = ("Michelangelo Caravaggio, Italian painter, is "
+     ...             "known for 'The Calling of Saint Matthew'.")
+     >>> document = client.document(text_val)
+     >>> document.analyze_entities()
+     {
+       'entities': [
+         {
+           'name': 'Michelangelo Caravaggio',
+           'type': 'PERSON',
+           'metadata': {
+             'wikipedia_url': 'http://en.wikipedia.org/wiki/Caravaggio'
+           },
+           'salience': 0.75942981,
+           'mentions': [
```

Comment:
It'd be nice if it was >0% abstracted :-)
Normalizing the casing of the response keys seems to be within gcloud-python's mission to make REST apis feel more pythonic, no?